src/SignalR/common/Protocols.MessagePack/src/Protocol/MessagePackHubProtocol.cs

```diff
@@ -358,247 +343,249 @@ private static object[] BindArguments(byte[] input, ref int offset, IReadOnlyLis
         /// <inheritdoc />
         public void WriteMessage(HubMessage message, IBufferWriter<byte> output)
         {
-            var writer = MemoryBufferWriter.Get();
+            var memoryBufferWriter = MemoryBufferWriter.Get();
 
             try
             {
+                var writer = new MessagePackWriter(output);
```

Comment:
```suggestion
                var writer = new MessagePackWriter(memoryBufferWriter);
```

Reply:
I don't know what to say, i was debugging few tests trying to understand why i had serialization issues.
I was in the process of testing small POC referencing 1.7 and 2.0 to see if i did not used a wrong `MessagePack` method itself 

Thx a lot for finding this typo